reco_utils/dataset/pandas_df_utils.py

```diff
@@ -73,7 +73,7 @@ def filter_by(df, filter_by_df, filter_by_cols):
     ]
 
 
-def libffm_converter(df, col_rating=DEFAULT_RATING_COL, filepath=None):
+class LibffmConverter():
```

Comment:
are we sure we need a class here? Normally classes in python are useful when we are saving state, do we need to do it here?

Reply:
This is to mimic the sklearn Binarizer and other similar stuff in the module of `preprocessing`, so mostly I followed their pattern. See [here](https://github.com/scikit-learn/scikit-learn/blob/7b136e9/sklearn/preprocessing/data.py#L1754).